frontend/src/pages/clusterSettings/PVCSizeSettings.tsx

```diff
@@ -34,34 +34,36 @@ all users."
       <Stack hasGutter>
         <StackItem>
           <InputGroup>
-            <TextInput
-              id="pvc-size-input"
-              style={{ maxWidth: '200px' }}
-              name="pvc"
-              data-id="pvc-size-input"
-              type="text"
-              aria-label="PVC Size Input"
-              value={pvcSize}
-              pattern="/^(\s*|\d+)$/"
-              onChange={async (value: string) => {
-                const modifiedValue = value.replace(/ /g, '');
-                if (modifiedValue !== '') {
-                  let newValue = Number.isInteger(Number(modifiedValue))
-                    ? Number(modifiedValue)
-                    : pvcSize;
-                  newValue =
-                    newValue > MAX_PVC_SIZE
-                      ? MAX_PVC_SIZE
-                      : newValue < MIN_PVC_SIZE
-                      ? MIN_PVC_SIZE
-                      : newValue;
-                  setPvcSize(newValue);
-                } else {
-                  setPvcSize(modifiedValue);
-                }
-              }}
-            />
-            <InputGroupText variant={InputGroupTextVariant.plain}>GiB</InputGroupText>
+            <InputGroupItem isFill>
+              <TextInput
+                id="pvc-size-input"
+                style={{ maxWidth: '200px' }}
+                name="pvc"
+                data-id="pvc-size-input"
+                type="text"
+                aria-label="PVC Size Input"
+                value={pvcSize}
+                pattern="/^(\s*|\d+)$/"
+                onChange={async (e, value: string) => {
+                  const modifiedValue = value.replace(/ /g, '');
+                  if (modifiedValue !== '') {
+                    let newValue = Number.isInteger(Number(modifiedValue))
+                      ? Number(modifiedValue)
+                      : pvcSize;
+                    newValue =
+                      newValue > MAX_PVC_SIZE
+                        ? MAX_PVC_SIZE
+                        : newValue < MIN_PVC_SIZE
+                        ? MIN_PVC_SIZE
+                        : newValue;
+                    setPvcSize(newValue);
+                  } else {
+                    setPvcSize(modifiedValue);
+                  }
+                }}
+              />
+            </InputGroupItem>
+            <InputGroupText>GiB</InputGroupText>
```

Comment:
And in `frontend/src/pages/clusterSettings/CullerSettings.tsx` -- same issue, I caught them both in the same screenshot and I'm too lazy to retake or to break up the screenshots 🙂 

Before:
<img width="1400" alt="Screenshot 2023-10-10 at 3 14 11 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/8126518/e040683a-5abe-4871-9e2f-621cb1fe7275">

After:
![Screenshot 2023-10-10 at 3 14 16 PM](https://github.com/opendatahub-io/odh-dashboard/assets/8126518/357e2f4d-e91a-444b-bd29-d1a715d2bcf6)